tools/ocaml_compiler_internal_params

```diff
@@ -0,0 +1,27 @@
+* : can-discard = ocamlcfg
```

Comment:
The `tools` directory doesn't seem like the right place for this file.  Maybe it should just live at the root?